deployment.yml

```
@@ -0,0 +1,67 @@
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14
+spec:
+  selector:
+    matchLabels:
+      app: module-14
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: module-14
+    spec:
+      containers:
+        - name: todo-app
+          image: todo-app:prod
+          #image: nginx
+          ports:
+          - containerPort: 80
```

This says 80 but your app actually seems to run on 8000 (which is also what your service assumes) so I think this needs updating:
```suggestion
          - containerPort: 8000
```